image-loader/test/scala/model/ProjectorTest.scala

```diff
@@ -117,37 +123,62 @@ class ProjectorTest extends FreeSpec with Matchers with ScalaFutures with Mockit
       identifiers = Map(),
       uploadInfo = UploadInfo(Some("getty.jpg")),
-      source = Asset(new URI("http://img-bucket.s3.amazonaws.com/i/d/1/2/3/id123"),
+      source = Asset(new URI("http://img-bucket.s3.amazonaws.com/i/d/1/2/3/" + id),
         Some(12666),
         Some(Jpeg),
         Some(Dimensions(100, 60)), None),
-      thumbnail = Some(Asset(new URI("http://thumb-bucket.s3.amazonaws.com/i/d/1/2/3/id123"),
-        Some(6404),
+      thumbnail = Some(Asset(new URI("http://thumb-bucket.s3.amazonaws.com/i/d/1/2/3/" + id),
+        Some(6397),
         Some(Jpeg),
         Some(Dimensions(256, 154)), None)),
       optimisedPng = None,
       fileMetadata = gettyFileMetadataExpected,
       userMetadata = None,
       metadata = ImageMetadata(
-        Some(new DateTime("2015-01-22T00:00:00.000Z").withZone(DateTimeZone.UTC)),
-        Some("Austria's Matthias Mayer attends the men's downhill training of the FIS Alpine Skiing World Cup in Kitzbuehel, Austria, on January 22, 2015.       AFP PHOTO / CHRISTOF STACHECHRISTOF STACHE/AFP/Getty Images"),
-        Some("AFP/Getty Images"),
-        None, Some("Christof Stache"), Some("Stringer"), None, Some("CHRISTOF STACHE"),
-        Some("-"), Some("AFP"), None, Nil, None, Some("Kitzbuehel"), None, Some("Austria"), List("sport")),
+        dateTaken = Some(new DateTime("2015-01-22T00:00:00.000Z").withZone(DateTimeZone.UTC)),
+        description = Some("Austria's Matthias Mayer attends the men's downhill training of the FIS Alpine Skiing World Cup in Kitzbuehel, Austria, on January 22, 2015.       AFP PHOTO / CHRISTOF STACHECHRISTOF STACHE/AFP/Getty Images"),
+        credit = Some("AFP/Getty Images"),
+        creditUri = None,
+        byline = Some("CHRISTOF STACHE"),
+        bylineTitle = Some("Stringer"),
+        title = Some("Austria's Matthias Mayer attends the men"),
+        copyright = Some("CHRISTOF STACHE"),
+        suppliersReference = Some("-"),
+        source = Some("AFP"),
+        specialInstructions = None,
+        keywords = Nil,
+        subLocation = None,
+        city = Some("KITZBUEHEL"),
+        state = Some("-"),
+        country = Some("AUSTRIA"),
+        subjects = List("sport"),
+        ),
       originalMetadata = ImageMetadata(
         Some(new DateTime("2015-01-22T00:00:00.000Z").withZone(DateTimeZone.UTC)),
         Some("Austria's Matthias Mayer attends the men's downhill training of the FIS Alpine Skiing World Cup in Kitzbuehel, Austria, on January 22, 2015.       AFP PHOTO / CHRISTOF STACHECHRISTOF STACHE/AFP/Getty Images"),
-        Some("AFP/Getty Images"), None, Some("Christof Stache"), Some("Stringer"),
-        None, Some("CHRISTOF STACHE"), Some("-"),
-        Some("AFP"), None, Nil, None, Some("Kitzbuehel"), None, Some("Austria"), List("sport")),
-      usageRights = Agency("Getty Images", Some("AFP"), None),
-      originalUsageRights = Agency("Getty Images", Some("AFP"), None),
+        Some("AFP/Getty Images"),
```

Comment:
can we do the named fields refactor to this?

Reply:
Done